packages/core-mobile/app/services/network/consts.ts

```diff
@@ -1,17 +1,89 @@
 import {
   AVALANCHE_XP_NETWORK,
   AVALANCHE_XP_TEST_NETWORK,
+  BITCOIN_NETWORK,
+  BITCOIN_TEST_NETWORK,
   ChainId,
   Network,
   NetworkVMType
 } from '@avalabs/core-chains-sdk'
 
+export const AVALANCHE_MAINNET_NETWORK = {
+  chainId: ChainId.AVALANCHE_MAINNET_ID,
+  chainName: 'Avalanche C-Chain/EVM',
+  description:
```

Comment:
for these hardcoded networks, let's only include the fields that are needed for the receive screen here. the less fields the better.